src/Avatar.svelte

```diff
@@ -1,25 +1,29 @@
 <script lang="ts">
-  import { onMount } from 'svelte';
   import { createIcon } from '@app/blockies';
   import { isAddress, isRadicleId } from '@app/utils';
 
+  export let address: string;
   export let source: string;
   export let inline = false;
   export let glowOnHover = false;
 
-  let container: HTMLElement;
+  function handleMissingFile() {
+    source = createContainer(address);
+  }
+
+  function createContainer(source: string) {
+    const seed = source.toLowerCase();
+    const avatar = createIcon({
+      seed,
+      size: 8,
+      scale: 16,
+    });
+    return avatar.toDataURL();
+  }
 
-  onMount(() => {
-    if (isAddress(source) || isRadicleId(source)) {
-      const seed = source.toLowerCase();
-      const avatar = createIcon({
-        seed,
-        size: 8,
-        scale: 16,
-      });
-      container.style.backgroundImage = `url(${avatar.toDataURL()})`;
-    }
-  });
+$: if (isAddress(source) || isRadicleId(source)) {
+    source = createContainer(source);
+  }
```

Comment:
Funky indent

Reply:
Yeah this is strange, I tried to indent it like:
```js
$: if (isAddress(source) || isRadicleId(source)) {
  source = createContainer(source);
}
```
But eslint objects with: `Expected indentation of 2 spaces but found 0`, we could disable this eslint rule for this line..

Reply:
Okay I think I found a nice solution, and eslint not objecting